src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -782,6 +782,9 @@ private async Task OrchestrationMiddleware(DispatchMiddlewareContext dispatchCon
 #pragma warning disable CS0618 // Approved for use by this extension
                         InvokeHandler = async userCodeInvoker =>
                         {
+                            // We yield control to ensure this code is executed asynchronously relative to WebJobs.
+                            // This ensures WebJobs is able to offload orchestrator code in the case of a timeout.

```

Comment:
nit: it's not clear what "offload orchestrator code" means. I suggest being more specific about what we're mitigating:
```suggestion
                            // This ensures WebJobs is able to correctly cancel the invocation in the case of a timeout.

```